smarts/core/smarts.py

```diff
@@ -1380,7 +1385,7 @@ def _try_emit_envision_state(self, provider_state: ProviderState, obs, scores):
                 )
                 traffic[v.vehicle_id] = envision_types.TrafficActorState(
                     actor_type=envision_types.TrafficActorType.SocialVehicle,
-                    vehicle_type=veh_type,
+                    vehicle_type=envision_types.VehicleType(veh_type),
```

Comment:
We will probably need to add a function to convert the types here.
However, I would suggest that it should be done in a different PR.

Reply:
(... the reason I suggest a different PR is because we've long needed to clean up our handling of vehicle types anyway and it's a bigger job than just this.)

Reply:
Is what is here okay for now though?

Reply:
@sah-huawei can you take another look at this please?

Reply:
As described above, and as we just chatted about, getting this right is going to take more than this.  Thus, I think it's best to just revert this line here to what it was originally.  I will open another issue about fixing vehicle types in general.